test/modules/TestResultCoordinator/service/TestResultEventReceivingService.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Service
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Common;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.EventHubs;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+
+    class TestResultEventReceivingService : BackgroundService
+    {
+        readonly ILogger logger = ModuleUtil.CreateLogger(nameof(TestResultEventReceivingService));
+
+        public override async Task StopAsync(CancellationToken stoppingToken)
+        {
+            this.logger.LogInformation("Test Result Event Receiving Service is stopping.");
+
+            await Task.CompletedTask;
+        }
+
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            this.logger.LogInformation("Test Result Event Receiving Service running.");
+
+            DateTime eventEnqueuedFrom = DateTime.UtcNow;
+            var builder = new EventHubsConnectionStringBuilder(Settings.Current.EventHubConnectionString);
```

Comment:
Why make this a var instead of EventHubsConnectionStringBuilder type?

Reply:
my general rule is to use var if you can determine the type from right side.